pkg/addressbook/addressbook.go

```diff
@@ -14,12 +14,11 @@ import (
 
 // KnownAddress represents additional manually crafted information about a particular account in the blockchain.
 type KnownAddress struct {
-	IsScam      bool `json:"is_scam,omitempty"`
-	RequireMemo bool `json:"require_memo,omitempty"`
-	// Name is a dns name.
-	Name    string `json:"name"`
-	Address string `json:"address"`
-	Image   string `json:"image,omitempty"`
+	IsScam      bool   `json:"is_scam,omitempty"`
+	RequireMemo bool   `json:"require_memo,omitempty"`
+	Name        string `json:"name"`
```

Comment:
it would be super helpful to explain what all these fields are used for, especially `Name`.

Reply:
ok.